### ISSUE TITLE
Unable to create a text generator, it throws error in numpy reshape.

### ISSUE THREAD

**Issue by JagathPrasad** [SciSharp/Keras.NET#158]
I have created the some sample for text generator using keras.net, while converting to np.reshape it throws error "Can't convert from NDarray[] to NDarray", please help me where I have done mistake.

            string text = System.IO.File.ReadAllText(text_path);
            string vocab = System.IO.File.ReadAllText(vocab_path);
            var charList = text.ToCharArray();
            var uniqueChars = vocab.ToCharArray();

            Dictionary<int, string> intToText = new Dictionary<int, string>();
            Dictionary<string, int> textToInt = new Dictionary<string, int>();
            int vocab_size = uniqueChars.Count();
            Utility.Utility utility = new Utility.Utility();
            textToInt = utility.ConvertTexttoInteger(uniqueChars);
            intToText = utility.ConvertIntegertoText(uniqueChars);

            List<Dictionary<string, int>> intList = new List<Dictionary<string, int>>();
            NDarray[] x = new NDarray[(charList.Count() - sequence_length)];
            NDarray[] y = new NDarray[(charList.Count() - sequence_length)];
            List<Dictionary<string, int>> intListOut = new List<Dictionary<string, int>>();
            for (int i = 0; i < (charList.Count() - sequence_length); i++)
            {
                char[] text_stringbuilder = new char[i + sequence_length];
                for (int j = i; j < i + sequence_length; j++)
                {
                    text_stringbuilder[j] = (charList[j]);
                }
                char[] text_out = new char[i + sequence_length];
                text_out[i] = charList[i + sequence_length];
                x[i] = utility.ConvertTexttoIntegerDynamic(text_stringbuilder, textToInt);
                y[i] = utility.ConvertTexttoIntegerDynamic(text_out, textToInt);

                if (i == 100) break;
            }

            var X = np.reshape(x[0], 5, 5, 1);
            X = X / uniqueChars.Count();

            var model = new Sequential();
            model.Add(new LSTM(256));
            model.Add(new Dropout(0.2));
            model.Add(new Dense(16, activation: "softmax"));
            model.Compile(optimizer: "adam", loss: "categorical_crossentropy");
            model.Fit(X, y, batch_size: 128, epochs: 50);
